tests/sdk/nodejs/examples/examples_test.go

```diff
@@ -455,6 +455,19 @@ func TestHelmReleaseCRD(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestHelmReleasePrometheus(t *testing.T) {
+	// Validate fix for https://github.com/pulumi/pulumi-kubernetes/issues/1710
+	skipIfShort(t)
+	test := getBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:         filepath.Join(getCwd(t), "helm-release-prometheus"),
+			SkipRefresh: false,
+			Verbose:     true,
+		})
+	// TODO validate the returned metadata for the prometheus rule in output.
```

Comment:
For some reason the get in the example is not returning information about the CR even though I have manually verified the appropriate CR exists. Ideally I would like to verify the same here. Will investigate and update the test accordingly.

Reply:
Could it be because `get` doesn't await the release operation correctly? Do we have no means to fetch resources from the `Release` resource itself? Or track its completion?